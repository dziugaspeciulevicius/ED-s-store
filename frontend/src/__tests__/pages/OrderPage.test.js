import React from "react";
import { shallow } from "enzyme";
import "../../setupTests";
import OrderPage from "../../pages/OrderPage.component";
jest.mock("../../pages/OrderPage.component");

describe("<OrderPage />", () => {
  it("Renders <OrderPage /> component correctly", () => {
    // shallow creates instance of a component
    expect(shallow(<OrderPage />)).toMatchSnapshot();
  });
});
