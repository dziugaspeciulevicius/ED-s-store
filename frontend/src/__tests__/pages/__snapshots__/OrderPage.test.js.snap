// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrderPage /> Renders <OrderPage /> component correctly 1`] = `ShallowWrapper {}`;
